libpod/runtime.go

```diff
@@ -1451,3 +1466,36 @@ func (r *Runtime) ImageRuntime() *image.Runtime {
 func (r *Runtime) SystemContext() *types.SystemContext {
 	return r.imageContext
 }
+
+// Since runc does not currently support cgroupV2
+// Change to default crun on first running of libpod.conf
+// TODO Once runc has support for cgroups, this function should be removed.
+func cgroupV2Check(configPath string, tmpConfig *RuntimeConfig) error {
+	if !tmpConfig.CgroupCheck && rootless.IsRootless() {
+		cgroupsV2, err := cgroups.IsCgroup2UnifiedMode()
+		if err != nil {
+			return err
+		}
+		if cgroupsV2 {
+			path, err := exec.LookPath("crun")
+			if err != nil {
+				// Can't find crun path so do nothing
+				return nil
```

Comment:
On the fence about having a debug statement in here too.  I'll let you make the call.

Reply:
Agree we should `Warnf` here about crun not existing - that's pretty serious